csodiaq/csodiaq_identification_functions.py

```diff
@@ -205,7 +215,7 @@ def traml_library_upload(fileName):
         lib_df = pd.read_csv(fileName)
     smf.print_milestone("Enter library dictionary upload: ")
 
-    # Pan human and spectraST libraries have different column names. This normalizes the columns.
```

Comment:
Minor: Pluralizing module/classes/etc. is always weird. I would go with "libraries" since it's grammatically correct and the context is clear, but that's just preference.

Reply:
I just realized that you used the replace function when you renamed the module, or refactoring went wonky. If it's the former, you can right-click files, then do Refactor -> Rename and Pycharm will do basically all the renaming for you.